public/journey-01.svg

```diff
@@ -0,0 +1,53 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="403" height="293" viewBox="0 0 403 293">
```

Comment:
We don't have to do this now but I think it might be useful to minify the SVGs in the future. We can probably use something like svgo to do that. This can possibly make the page load faster, but it's not a big deal right now, we can do this later. https://github.com/svg/svgo

Reply:
I made the change and added the minify step to the autofix script.